### PR TITLE
Grant 107 Submission Table in dashboard

### PR DESCRIPTION
**Pull request by PeterPavlatos**
Dashboard: Created admin dashboard table with dummy submission data.
Search:        User is currently able to search data in 1 column for now.
Pagination:  User is able to move to next, previous and first and last page of data.

image

### CODE REVIEW

File: client/pages/form-pages/application-table.tsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
+import { Button, SearchBar } from '@components';
+import { Pagination } from '../../components/form/Pagination';
+import { useState } from 'react';
+import initialValues from './example_submission_data.json';
+
+export interface ActivitiesGapProps {
```
Comment (karthikktamilmani-freshworks) [inline client/pages/form-pages/application-table.tsx:6]: Can be removed along with the commented-out things!

File: client/pages/form-pages/application-table.tsx [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    <tbody>
+      {applications.applications &&
+        applications.applications.map((row: any, index: number) => (
+          <>
+            <tr
+              key={`row${index}`}
+              onClick={() => handleSelectRow(row.BCAAP_Form_ID)}
+              className='bg-white border-b table-row-fixed'
+            >
+              <td className={tdStyles}>{row.Confirmation_ID}</td>
```
Comment (karthikktamilmani-freshworks) [inline client/pages/form-pages/application-table.tsx:55]: Could you also make sure that the styling, colors align with the design!

File: client/pages/form-pages/application-table.tsx [hunk 3]
```diff
@@ -0,0 +1,151 @@
+import { Button, SearchBar } from '@components';
```
Comment (karthikktamilmani-freshworks) [inline client/pages/form-pages/application-table.tsx:1]: Could you move this inside the components sections!

File: client/components/forms/FormContent.tsx [hunk 4]
```diff
@@ -19,7 +20,7 @@ export const FormContent: React.FC<FormContentProps> = ({ step, formTitle }) =>
   const showStepContent = () => {
     switch (formTitle) {
       case FormSteps.CONTACT_INFO:
-        return <ContactInfo />;
+        return <ApplicationPage />;
```
Comment (karthikktamilmani-freshworks) [inline client/components/forms/FormContent.tsx:23]: I don't think this update is needed

File: client/pages/form-pages/example_submission_data.json [hunk 5]
```diff
@@ -0,0 +1,2606 @@
+[
```
Comment (karthikktamilmani-freshworks) [inline client/pages/form-pages/example_submission_data.json:1]: Could you move this inside the constants?

**Review by karthikktamilmani-freshworks — CHANGES_REQUESTED** [review]
Good job overall! Also, make sure you rebase the branch with develop!

File: client/pages/form-pages/application-table.tsx (hunk 1 above)
Comment (PeterPavlatos) [reply client/pages/form-pages/application-table.tsx]: Removed commented out code.

File: client/pages/form-pages/application-table.tsx (hunk 2 above)
Comment (PeterPavlatos) [reply client/pages/form-pages/application-table.tsx]: OK

**Review by karthikktamilmani-freshworks — CHANGES_REQUESTED** [review]
Could you make sure that the UI matches the color, format and size of the designs?

**Pull request merged**